### DISCUSSION TITLE
Fastest IP address mode, client can not receive the ip address of special host

### DISCUSSION THREAD

**Discussion by xiao-mantou** [AdguardTeam/AdGuardHome#3206] [Q&A] [answered]
### Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.106.3
* **How did you install AdGuard Home:**
  * Github releases
* **How did you setup DNS configuration:**
  * System
* **CPU architecture:**
  * AMD64
* **Operating system and version:**
  * Windows 10 2004

### Actual Behavior
Mode: Fastest IP Address
Domain name has been resolved, but client didn't receive it.
`Ping request could not find host lightswitch-public-service-prod06.ol.epicgames.com. Please check the name and try again.`

### Screenshots

<details><summary>Screenshot:</summary>

![AdGuardHome](https://user-images.githubusercontent.com/25640395/120072915-981aab00-c085-11eb-83fb-25c67f8ccebc.png)
![Ping](https://user-images.githubusercontent.com/25640395/120072957-c5675900-c085-11eb-9ad1-05b5c82628e7.jpg)
![Epic](https://user-images.githubusercontent.com/25640395/120073013-0f503f00-c086-11eb-98c5-df0089bb436a.png)

</details>

### Additional Information

Fastest IP Address mode Only.

**Comment by ameshkov** [comment, marked as answer]
Most likely that's because elapsed time is just too high (~10 seconds).

Next queries should be faster since the response should be already cached.

You can also check out AdGuad logs to find out what exactly takes that much time. Here's how to [configure](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) AdGuard Home to write verbose-level log.
